docs/content/en/docs/user-guide/automated-deployment-analysis.md

```diff
@@ -8,21 +8,40 @@ description: >
 
 >NOTE: This feature is currently alpha status.
 
-Automated Deployment Analysis (ADA) lets you automate the verification of the deployment process by analyzing the metrics data, log entries, and the responses of the configured HTTP requests.
+Automated Deployment Analysis (ADA) evaluates the impact of the application you are about to deploy by analyzing the metrics data, log entries, and the responses of the configured HTTP requests.
```

Comment:
> `impact of the application`

Looks like `impact of deployment` is more correct because here we are talking about the impact of the deployment on the application.

Reply:
True